mod/config/pkg/spec/special_cases.go

```diff
@@ -28,3 +30,9 @@ const (
 	//nolint:lll // temporary.
 	BArtioValRoot = "0x9147586693b6e8faa837715c0f3071c2000045b54233901c2e7871b15872bc43"
 )
+
+const (
+	BoonetFork1Height uint64 = 69420
+
+	BoonetFork2Height uint64 = math.MaxUint64
```

Comment:
@calbera I think we should temporarily list Boonet forks here, just to avoid spreading them across the codebase